roles/activemq/meta/argument_specs.yml

```diff
@@ -0,0 +1,14 @@
+---
+argument_specs:
+  main:
+    short_description: Activemq
+    author: Alfresco
+    options:
+      activemq_version:
+        description: activemq version to use for deployment.
+        required: false
+        default: 5.17.4
+      activemq_log_config:
+        description: activemq log4j and log4j2 config file.
+        required: false
+        default: log4j2
```

Comment:
This should be a role argument only if it allow you to configure activemq to use log4j or log4jv2 regardless of the version but that's not possible.
As this variable is not used anywhere else but in a single task would make it a task variable only, not a role argument.